### PR TITLE
Update version of pug

### PR DESCRIPTION
**Pull request by millerds**
Update to fix vulnerability

### CODE REVIEW

**Review by TCourtneyOwen — COMMENTED** [review]
Actually we probably only want to include the package.json and package-lock.json change in this change.

**Comment by akrantz** [comment]
@TCourtneyOwen What is pug for?

File: packages/eslint-config-office-addins/package-lock.json [hunk 1]
```diff
@@ -1,1309 +1,1309 @@
...
-      }
-    },
-    "yargs-parser": {
-      "version": "20.2.6",
-      "resolved": "https://registry.npmjs.org/yargs-parser/-/yargs-parser-20.2.6.tgz",
-      "integrity": "sha512-AP1+fQIWSM/sMiET8fyayjx/J+JmTPt2Mr0FkrgqB4todtfa53sOsrSAcIrJRD5XS20bKUwaDIuMkWKCEiQLKA==",
-      "dev": true
-    }
-  }
+	"name": "eslint-config-office-addins",
```
Comment (akrantz) [inline packages/eslint-config-office-addins/package-lock.json:2]: Why is the indent level different?

**Review by akrantz — CHANGES_REQUESTED** [review]
Could you see about making the indent level the same so there aren't a bunch of whitespace changes?

File: packages/eslint-config-office-addins/package-lock.json (hunk 1 above)
Comment (millerds) [reply packages/eslint-config-office-addins/package-lock.json:2]: That was automatic . . . I'll have to manually take a look.

**Comment by TCourtneyOwen** [comment]
pug is used in setting up the express server, although I can't recall why this is needed - this code came from the original template Ricky made.  https://www.npmjs.com/package/pug. 

Interestingly though, I just commented out this line from office-addin-sso (the only place pug is used), built, and linked to the Office-Addin-SSO template, and everything works fine:  this.appInstance.set('view engine', 'pug');

So, we could probably just remove pug and remove this like from %:\Office-Addin-Scripts\packages\office-addin-sso\src\app.ts

**Comment by akrantz** [comment]
I vote to remove pug...

**Pull request closed without merging**